src/remote.sh

```diff
@@ -48,13 +48,21 @@ function cmd_remotely()
   # Set default values if not specified
   remote=${remote:-"localhost"}
   port=${port:-"22"}
-  user=${user:-"root"}
+  user=${user:-$(whoami)}
```

Comment:
This assigns the username to a user from the local machine, which may not be the same from the remote machine, I guess.

Reply:
Correct!
but this assignment happens if and only if no other user name has been provided. 
I only changed the default value from root to the username of the current user which I think is the best guess we could make. If such a user does not exist then an username would have to be provided in the config file anyway.

Currently the only way to provide an user in kw is through the ssh_ip field in the config file, which is a whole other can of worms that I didn't want to mess with right now haha

Reply:
#358 addresses this user issue! Maybe this could be rebased of that...